### PR TITLE
fix missing link style in admin.scss

### LINKED ISSUE #1998: No default styling on links (that are not UI elements) in admin panel
### Steps to reproduce the problem

1. In the admin panel, issue a warning to a user that contains a link.
2. Open the warning and note that it is in the link-blue.
 ~~This can be replicated with links in mod notes and reports too.~~ This is not an issue with mod notes and reports, as they do not automatically hyperlink.
...

### Expected behaviour

Link styling should be consistent with the rest of the theme.

### Actual behaviour

Default link-blue is used, which is incredibly hard to read on the dark glitch theme

### Detailed description

This seems to be due to the lack of a default `a` style in the `.admin_wrapper` element of `app/javascript/flavours/glitch/styles/admin.scss`.

### Specifications

4.0.1+glitch
Ruby 3.0.4p208
NodeJS v18.12.0

Vivaldi 5.6.2867.36

### PR DESCRIPTION
**Pull request by AtelierSnek**
Fixes #1998,  however a more specific style may be desirable, so happy to adjust this however to make it consistent with the rest of the style.

### CODE REVIEW

**Comment by Plastikmensch** [comment]
This also changes other link elements:
Screenshot_2022-12-08_06-03-19
(Default, link to user is grey)

Screenshot_2022-12-08_06-04-00
(After applying rule in PR, link to user is blue)

It's probably better to only apply the rule to `.strike-card` or `.report-header__card`

**Comment by AtelierSnek** [comment]
> It's probably better to only apply the rule to `.strike-card` or `.report-header__card`

Both, or just one?

**Comment by AtelierSnek** [comment]
Hmm, looking at the styling:
image
and keeping it to the most specific element, it's `.strike-card`. So will add to that.

**Comment by Plastikmensch** [comment]
Unfortunately this issue seems to be shared with upstream, so a similar change is needed for the vanilla flavour.
Using glitch flavour:
Screenshot_2022-12-08_06-35-59
(correct blue color used for links)

Using vanilla flavour:
Screenshot_2022-12-08_06-34-36
(unreadable default color is used)

**Comment by Plastikmensch** [comment]
I think one thing to make it more consistent with other link elements is to add `text-decoration: none;` to `a` and change it in `&:hover` to `text-decoration: underline;`, basically reversing the current behaviour.
Maybe also removing `&:active` and `&:focus` as no other link elements use these.

TL;DR:
```css
a {
  color: $highlight-text-color;
  text-decoration: none;

  &:hover {
    text-decoration: underline;
  }
}
```

**Review by Plastikmensch — APPROVED** [review]

**Comment by ClearlyClaire** [comment]
Hi! This is an issue we share with upstream, would you mind submitting it there instead?

**Comment by AtelierSnek** [comment]
> Hi! This is an issue we share with upstream, would you mind submitting it there instead?

Sure! There are two issues, to be clear, both the glitch flavour and the default mastodon flavour have this issue. But will submit this upstream in the morning :)
